content/docs/intro/tutorial/adding-pages/index.md

```diff
@@ -27,7 +27,7 @@ Taking a look at `src/app/app.html`, we see this line near the bottom:
 
 Pay attention to the `[root]` property binding. This sets what is essentially the first, or "root" page for the `ion-nav` component. When `ion-nav` loads, the component referenced by the variable `rootPage` will be the root page.
 
-In `src/app/app.component.ts`, the `MyApp` component specifies this in its constructor:
+In `src/app/app.component.ts`, the `MyApp` component specifies the HelloIonicPage as the root page by assigning it to the rootPage variable:
```

Comment:
I think it would sound better without `the` and look better rendered with ` around HelloIonicPage.